packages/harmony/src/components/layout/Box/Box.tsx

```diff
@@ -51,8 +53,8 @@ export const Box = styled.div<BoxProps>(
     return {
       position: 'relative',
       boxSizing: 'border-box',
-      height: h,
-      width: w,
+      height: spacing[h as SpacingOptions] ?? h,
```

Comment:
Not 100% sold on the idea of spacing vars used for things like height and width generally. I think i would prefer things like icons having their own sizing vars and using that as opposed to using the spacing vars, but not strongly enough to block the PR or anything. down to chat about this tomorrow 👍

Reply:
yeah i definitely agree with your thoughts here, this was a bit of a test-case to see what folks thought... lets def chat about this. given that you and jd are out, im going to merge if that's okay so i can get on to other items, but ill be sure we talk as as a group and figure out the plan